### ISSUE TITLE
Way longer routing than needed (may caused by the landslide hapend there?)

### ISSUE THREAD

**Issue by Mannshoch** [osmandapp/OsmAnd#20451]
### Routing engine

- [X] OsmAnd's in-app offline routing
- [ ] Any online routing provider (YOURS, openrouteservice, OSRM, etc.)

### Routing Profile

Auto

### Start and end points

46.22297N, 9.05358E to 46.72346N, 943992E

### Actual route

At some point it left the motorway and route a way longer route thought the vilages.
I'm not sure if the Problem came from a close by an earth slide, but the motorway is open again. I used it last week.

### Expected route

Completely on the motorway

### Maps you used (online or offline)

- [ x ] Offline maps offered within the OsmAnd app for download. - 01.07.2024
- [ ] Online (tile / raster) maps 

### Your Environment

```
OsmAnd Version: 4.7.17
Android version: 20
Device model: Pixel 5

```

### Anything else?

If the routing Problem came because of the landslide. I assume a message and the possibility to deactivate this blocking.

**Comment by yuriiurshuliak** [comment]
The bug was not reproduced. To reproduce this bug, build navigation using the specified coordinates on the reset car profile. 
I compared  the built route on the web version, OSM, and within the app. The route is consistent across all platforms and covers 99.4 km.

Please try updating the map and app, and reset the profile settings via Menu -> Settings -> App profile (Driving) -> Reset to default.

Web version link: [Navigation](https://test.osmand.net/map/navigate/?start=46.222973,9.053579&end=46.723462,9.439913&profile=car#9/46.5519/8.7177)

OSM link: [Navigation](https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=46.22302%2C9.05356%3B46.72346%2C9.43991#map=9/46.4938/9.0829)

OsmAnd~ 4.9.0#2872m, released: 2024-07-31

<table style="width:100%">
  </tr>
  <tr>
    <th>OSM</th>
    <th>Web version</th>
    <th>Mobile app</th>
  </tr>
  <tr>
<th><img src="https://github.com/user-attachments/assets/0535992d-977b-489d-8ed7-c9f4f51b061d"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/efe06786-548e-4abe-a11b-1fcc08708851"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/4040aeb0-a7b0-44d5-9486-043c8adef0c6"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by vshcherb** [comment]
Probably it was fixed in 4.8
